subsys/bluetooth/host/Kconfig

```diff
@@ -4,14 +4,29 @@
 # Copyright (c) 2015-2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config BT_LONG_WQ
+	# Enable dedicated wq for long-running tasks
+	bool "Enable dedicated workqueue for long-running tasks."
+	default y if BT_GATT_CACHING || BT_TINYCRYPT_ECC
+
```

Comment:
You need to add a proper help text here, explaining what happens if this gets disabled, i.e. that the operations will run in system work queue context then and have some potentially unexpected (bad) behaviour.